packages/react/src/profile/useCreateProfile.ts

```diff
@@ -29,31 +32,63 @@ export type CreateProfileOperation = Operation<
  * @category Profiles
  * @group Hooks
  *
- * @example
+ * @example Simple usage
  * ```tsx
  * import { useCreateProfile } from '@lens-protocol/react-web';
  *
  * function CreateProfile() {
+ *   const { error, execute, isPending } = useCreateProfile();
+ *
+ *   const onClick = async () => {
+ *     const handle = window.prompt("Enter your handle");
+ *
+ *     const result = await execute({ handle });
+ *
+ *     if (result.isSuccess()) {
+ *       console.log("Profile created!");
+ *     }
+ *   };
+ *
+ *   return (
+ *     <div>
+ *       { error && <p>{error.message}</p>}
+ *       <button disabled={isPending} onClick={onClick}>
+ *         Create profile
+ *       </button>
+ *     </div>
+ *   );
+ * }
+ * ```
+ *
+ * @example Programmatic error handling
+ * ```tsx
+ * import { useCreateProfile, DuplicatedHandleError } from '@lens-protocol/react-web';
```

Comment:
should example in `react` package mention import from `react-web` package?

Reply:
I see, maybe let's just leave it as it is then, till we have a separate package for react-native, then we will have to figure it out